packages/sdk/src/evm/constants/addresses/getContractPublisherAddress.ts

```diff
@@ -1,13 +1,13 @@
+import { getProcessEnv } from "../../../core/utils/process";
+
 const ContractPublisher_address = "0x664244560eBa21Bf82d7150C791bE1AbcD5B4cd7"; // Polygon only
 
 /**
  * @internal
  */
 export function getContractPublisherAddress() {
-  // eslint-disable-next-line turbo/no-undeclared-env-vars
-  if (process.env.contractPublisherAddress) {
-    // eslint-disable-next-line turbo/no-undeclared-env-vars
-    return process.env.contractPublisherAddress as string;
+  if (getProcessEnv("contractPublisherAddress")) {
+    return getProcessEnv("contractPublisherAddress");
```

Comment:
this looks weird, why not pass the `ContractPublisher_address` below to the default value you added to the getEnv?

Reply:
I like it